src/client/lib/_mapbox/_mapbox.js

```diff
@@ -0,0 +1,11 @@
+import 'mapbox-gl/dist/mapbox-gl.css'
```

Comment:
Why are the underscores? I'd change the filename to `lib/mapbox/mapbox.js`

Reply:
The idea was to don't get people confused on which mapbox they import but I can change this